lib/travis/build/script/dart.rb

```diff
@@ -236,6 +236,10 @@ def dartfmt
             end
 
             sh.cmd 'unformatted=`dartfmt -n .`'
```

Comment:
`$unformatted` is set at run time by this assignment.

If you want to trim it, you'd have to come up with a `bash` way to trim it (with `tr`, `grep`, `head`, `tail`, `sed`, whatever).